inc/widgets/post-loop.php

```diff
@@ -252,10 +252,11 @@ function form( $instance ) {
 					<?php foreach($templates as $template) : ?>
 						<option value="<?php echo esc_attr($template) ?>" <?php selected($instance['template'], $template) ?>>
 							<?php
-							$headers = get_file_data( locate_template($template), array(
```

Comment:
Here again, are we using a different mechanism somewhere to allow child themes to override post loop templates?